.circleci/config.yml

```diff
@@ -134,6 +139,18 @@ jobs:
       - save_workspace
 
 
+  deploy:
+    executor: node
+    steps:
+      - load_workspace
+      - run:
+          name: Configure NPM authentication
+          command: npm config set "//registry.npmjs.org/:_authToken" "$NPM_TOKEN"
+      - run:
+          name: Publish package
+          command: lerna publish from-package --yes --force-publish --no-git-tag-version --no-push --no-verify-access --dist-tag next
```

Comment:
- I don't see any documentation for the `--force-publish`, `--no-git-tag-version`, `--no-push` flags, what are they doing?
- Based on the documentation I think we should be able to get rid of `--no-verify-access`.

Reply:
> I don't see any documentation for the --force-publish, --no-git-tag-version, --no-push flags, what are they doing?

`lerna publish` wraps `lerna version` so you can pass it the same flags.

> Based on the documentation I think we should be able to get rid of --no-verify-access.

I think you're right. I wrongly assumed that this was to bypass the OTP and copied it over from what we are currently using. Let's try it!